test-cases/README.md

```diff
@@ -20,6 +20,8 @@ Gutenberg:
 - [Verse block](./gutenberg/verse.md)
 - [Video block](./gutenberg/video.md)
 - [File block](./gutenberg/file.md)
+- [Audio block](./gutenberg/audio.md)
```

Comment:
@jd-alexander I realized that the Audio block wasn't included in this index so went ahead and added it, let me know if it's ok.

Reply:
@fluiddot Thanks for adding it here. 👍🏾